examples/runFasterCode.ts

```diff
@@ -28,12 +28,8 @@ function getFromEnvOrThrow(configField: string): string {
 }
 
 const token = getFromEnvOrThrow('MARINER_GITHUB_TOKEN');
-const inputFilePath =
-    process.env.MARINER_INPUT_FILE_PATH ||
-    path.join(__dirname, '..', '..', 'examples', 'exampleData.json');
-const outputFilePath =
-    process.env.MARINER_OUTPUT_FILE_PATH ||
-    path.join(__dirname, '..', '..', 'examples', 'output.json');
+const inputFilePath = `${__dirname}${config.inputFilePath}`;
```

Comment:
I don't think we want to refer to __dirname at all. Just set our inputFilePath to whatever config.inputFilePath is. 

Rationale: A user might want to use an absolute path (like /etc/mariner/config.json) rather than having it be relative to the current directory or whatever directory the code is in.